msvc/mono.winconfig.targets

```diff
@@ -267,7 +296,7 @@
 
                     CreateVersionFile (versionFile);
 
-                    Log.LogMessage (MessageImportance.High, "Successfully setup Mono configuration headers.");
+                    Log.LogMessage (MessageImportance.High, "Successfully setup Mono configuration headers {0} {1} from {2}.", configFile, versionFile, winConfigFile);
```

Comment:
put an `and` between `{0}` and `{1}`

Reply:
And then if the list gets larger have to quibble over the syntax, but ok.

Reply:
Btw i tried the $”{foo}” thing, didn’t work right away, I gave up quick.